### PR TITLE
Edit tool tips and html

### PR DESCRIPTION
**Pull request by privatezero**
Addresses https://github.com/MediaArea/BWFMetaEdit/issues/32
* Changed `OriginatorReference` FADGI examples in tool tip and html to reflect FADGI documentation.
* Changed `Description` tool tip to strip erroneously placed `IARL` information and to reflect FADGI documentation.
* Changed `Description` html to strip erroneous placed `IARL` information.
* Changed `Originator` tool tip and html to reflect automatic copying to `IARL`.

Also, looks like my linux editor stripped out some carriage returns in the html file?

### CODE REVIEW

File: Source/Resource/Documentation/bext.html [hunk 1]
```diff
@@ -53,7 +52,10 @@
...
+					<li><p>This element contains the entity responsible for the "archiving" (creation, maintenance, preservation) of this digital item. Entity designations should be as specific as possible including a two-character county code to avoid the potential for conflict in the archiving entity’s name.</p>
+<p>If space permits within the 32 character limit, the archival entity should be identified at the most specific level within the institution.</p>
+<p>Use a standard abbreviation of entity names such as those found in the Guide to Government Acronyms & Abbreviations. If an entity is not on this list, use a familiar abbreviation. Use the standard two-character ISO 3166 alpha 2 country code list.</p>
```
Comment (JeromeMartinez) [inline Source/Resource/Documentation/bext.html]: `&` must be HTML escaped (`&amp;`)

**Comment by JeromeMartinez** [comment]
> Also, looks like my linux editor stripped out some carriage returns in the html file?

Looks like the previous code was a bit messy :). I think it is fine as is.

File: Source/Resource/Documentation/bext.html (hunk 1 above)
Comment (privatezero) [reply Source/Resource/Documentation/bext.html]: oops! Changed it!

Comment (privatezero) [reply Source/Resource/Documentation/bext.html]: oh except I made a typo! will fix

File: Source/Resource/Documentation/bext.html [hunk 2]
```diff
@@ -54,7 +54,7 @@
 				<ul>
 					<li><p>This element contains the entity responsible for the "archiving" (creation, maintenance, preservation) of this digital item. Entity designations should be as specific as possible including a two-character county code to avoid the potential for conflict in the archiving entity’s name.</p>
 <p>If space permits within the 32 character limit, the archival entity should be identified at the most specific level within the institution.</p>
-<p>Use a standard abbreviation of entity names such as those found in the Guide to Government Acronyms & Abbreviations. If an entity is not on this list, use a familiar abbreviation. Use the standard two-character ISO 3166 alpha 2 country code list.</p>
+<p>Use a standard abbreviation of entity names such as those found in the Guide to Government Acronyms &amp Abbreviations. If an entity is not on this list, use a familiar abbreviation. Use the standard two-character ISO 3166 alpha 2 country code list.</p>
```
Comment (JeromeMartinez) [inline Source/Resource/Documentation/bext.html]: Sorry, but `&amp;`, including semicolon. See e.g. [W3C example](https://www.w3.org/International/questions/qa-escapes#use)

Comment (retokromer) [inline Source/Resource/Documentation/bext.html]: Should read `&amp;` with a semicolon at the end.

Comment (privatezero) [reply Source/Resource/Documentation/bext.html]: Caught the typo after I commited! Still haven't had enough coffee this morning! Just fixed it

**Comment by dericed** [comment]
Thanks, this looks good. Also ping to @kmurmur on this one.

**Comment by JeromeMartinez** [comment]
thanks!

**Pull request merged**